doc/source/user_guide/io.rst

```diff
@@ -5593,164 +5593,164 @@ Given the next test set:
 
 .. code-block:: python
 
-   from numpy.random import randn
+    from numpy.random import randn
 
-   sz = 1000000
-   df = pd.DataFrame({'A': randn(sz), 'B': [1] * sz})
+    sz = 1000000
```

Comment:
can you add a random seed

Reply:
Ok, I will wait for all pipeline checks to pass then add it.

Reply:
Done